### PR TITLE
added animepill.com

### PR DESCRIPTION
**Pull request by Venipa**
Website: [animepill.com](animepill.com)

image

### CODE REVIEW

**Comment by Venipa** [comment]
how does updating metadata cause these codacy errors...

**Comment by Venipa** [comment]
fyi @Timeraa @Bas950

File: websites/A/AnimePill/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "author": {
+    "name": "Cascade",
+    "id": "442612628104151040"
+  },
+  "service": "AnimePill",
+  "description": {
+    "en": "Watch the latest anime hits. AnimePill - your daily dose of anime & manga."
+  },
+  "url": "animepill.com",
+  "version": "1.0.2",
```
Comment (ririxidev) [inline websites/A/AnimePill/dist/metadata.json:12]: 1.0.0 on release

File: websites/A/AnimePill/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "logo": "https://i.mavis.moe/f/wfFcEQf0Of/unbewhmi-waifu2x-art-noise1-scale-tta-1-benutzerdefiniert-png.png",
+  "thumbnail": "https://i.mavis.moe/f/vpOrz4JxUA/2020-11-18-18-14-01png.png",
```
Comment (ririxidev) [inline websites/A/AnimePill/dist/metadata.json:14]: upload to imgur and use i.imgur

File: websites/A/AnimePill/presence.ts [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    }
+  }
+],
+ presence = new Presence({
+  clientId: "778672856347312129"
+});
+
+let currentVideo: VideoContext;
+
+const initialize = async () => {
```
Comment (ririxidev) [inline websites/A/AnimePill/presence.ts:95]: for what reason you are initializing because events works every 1 seconds or something :)

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Comment by Venipa** [comment]
@ririxidev should be fine now (requested changes have been made)

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**